apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net/GoogleStackdriverAppender.cs

```diff
@@ -238,8 +233,13 @@ private LoggingServiceV2Client BuildLoggingServiceClient() =>
         private GoogleCredential GetCredentialFromConfiguration()
         {
             var credential =
+// Temporarily disable warnings for obsolete methods. See
+// https://github.com/googleapis/google-api-dotnet-client/pull/3043
```

Comment:
Point to the Spanner follow up issue now (which will be Spanner and Log4Net and NLog). If you haven't created it yet, create one with minimum details that we can expand on later.